websites/A/AnimeX.tech/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "author": {
+    "name": "NotMarek",
+    "id": "206299359007080450"
+  },
+  "service": "AnimeX",
+  "description": {
+    "en": "AnimeX is a free anime streaming site.",
+    "ro": "AnimeX este un site de streaming gratuit pentru anime.",
+    "pt_BR": "AnimeX é um site de streaming de animes gratuito.",
+    "fr": "AnimeX est un site de streaming d'animés gratuit.",
+    "uk_UA": "AnimeX - безкоштовний веб-сайт для стрімінгу аніме.",
+    "nl": "AnimeX is een gratis anime streaming-site."
```

Comment:
I bet you dont speak all these languages, only add descriptions for the languages you can speak.